f3/granite.go

```diff
@@ -92,12 +99,19 @@ type SignedMessage struct {
 	Value ECChain
 }
 
-// Aggregated list of GossiPBFT messages with the same instance, round and value. Used as evidence for justification of messages
-type QuorumSignature struct {
-	// Indexes in the base power table of the signers (bitset)
-	Signers bitfield.BitField
-	// BLS aggregate signature of signers
-	Signature []byte
+func (p Payload) MarshalForSigning(nn NetworkName) ([]byte, error) {
```

Comment:
Could we not get rid of using `.Current` everywhere if we just make this a method of `GMessage` and create a new method of `GMessage` `createJustification` that takes the signers and signatures as parameters returns the Justification? I think this would be my preferred approach.